phase1/vsphere/README.md

```diff
@@ -75,40 +72,50 @@ If you do power it on, future clones of this VM will end up [getting the same IP
 echo -n > /etc/machine-id
 ```
 
+### Launch deployment environment
 
-### Download `kubernetes-anywhere`:
+#### Pull `cnastorage/kubernetes-anywhere:latest` docker image:
 
 ```shell
-curl -sL https://github.com/kubernetes/kubernetes-anywhere/archive/v0.1.0.tar.gz | tar xz
-cd kubernetes-anywhere-0.1.0
+docker pull cnastorage/kubernetes-anywhere
 ```
 
-### Enter the `kubernetes-anywhere` deployment environment:
+#### Run docker container using `cnastorage/kubernetes-anywhere:latest` image to launch deployment environment:
 
 ```shell
-make docker-dev # Building docker image for first time can take few minutes.
+docker run -it --rm --env="PS1=[container]:\w> " --net=host cnastorage/kubernetes-anywhere:latest /bin/bash
 ```
 
-### Start the deployment wizard:
+#### Start the deployment wizard:
 
-```shell
-make deploy
-```
+### Sample config
 
-and fill complete the config wizard to deploy a kubernetes-anywhere cluster.
+Lets take a look at vSphere environment before starting deployment wizard.
 
-### Sample config
 
-**You can get help for any config option by entering '?'.** 
+![alt text](https://user-images.githubusercontent.com/22985595/27245213-0ff5654c-529f-11e7-8312-74b64707994c.jpg)
+
+Here you see Datacenter ```PA-DC```, has ```cluster-vsan-1``` with one resource pool - ```dev-resource-pool```.
+
+Following steps will deploy 4 nodes Kubernetes Cluster on the ```dev-resource-pool``` using the template file deployed on the cluster.
```

Comment:
This is just a question. Is this "resource pool" thing a new requirement of kubernetes-anywhere? I think before I never set it up... And it's not default in my nimbus testbed... Just wondering if I have to set it up if I want to use k8s-anywhere now.